rsconnect/main.py

```diff
@@ -244,6 +244,58 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+# inverts bool args if they are provided, otherwise returns None
```

Comment:
Doesn't just invert the bool args, but also sets them from the shorthand/"both" flag if that's present.